src/pages/access_review/AccessReviewHeader.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { div, img, hh } from 'react-hyperscript-helpers';
 import { Theme } from '../../libs/theme';
 import lockIcon from '../../images/lock-icon.png';
+import * as fp from 'lodash/fp';
```

Comment:
We haven't officially decided as a team what to do here, but I think the consensus is moving towards only importing the specific methods you want to use. In this case, just `isNil`